.github/workflows/Pipeline.yml

```diff
@@ -1,450 +1,116 @@
 name: Unit Testing, Coverage Collection, Package, Release, Documentation and Publish
 
-on: [ push ]
-
-defaults:
-  run:
-    shell: bash
+on: 
```

Comment:
I prefer `on: [push, workflow_dispatch]`, Yours creates 2 dictionary entries instead of 2 strings in a list

Reply:
Discussed and agreed in a meeting.